cypress/integration/brewery/CreateScenario-tests.spec.js

```diff
@@ -44,8 +44,8 @@ describe('Create scenario', () => {
     Login.login();
     // Create "another scenarios"
     Scenarios.createScenario(otherScenarioName, true, DATASET.BREWERY_ADT, RUN_TEMPLATE.BREWERY_PARAMETERS).then(
-      (value) => {
-        otherScenarioId = value.scenarioCreatedId;
+      (req) => {
+        otherScenarioId = req.scenarioCreatedId;
```

Comment:
I don't know if `req` is more accurate here or improves readability :sweat_smile: 
IIUC `req` contains the result of the call to `createScenario`. Maybe we could call it `result` or `createdScenario` ? (although `createdScenario.scenarioCreatedId` does not sound good, we may have to change the name of the object keys)